plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/transform/ArrowFunction_Part1_Transformation.xtend

```diff
@@ -14,12 +14,15 @@ import com.google.inject.Inject
 import org.eclipse.n4js.n4JS.ArrowFunction
```

Comment:
for transpiling arrow functions, two steps are required and inbetween something else has to be done (i.e. two transformations are required that need to happen at different points in the transpilation pipeline)